zulip_bots/zulip_bots/bots/merels/merels.py

```diff
@@ -0,0 +1,29 @@
+from zulip_bots.bots.merels.libraries import database
+from zulip_bots.bots.merels.libraries import game
+
+DATABASE_NAME = "merels_database.db"
+
+
+class MerelsBot(object):
+    """
+    Simulate the merels game to the chat
+    """
+
+    def __init__(self):
+        self.merels_db = database.MerelsDatabase(DATABASE_NAME)
+
+    def usage(self):
+        return game.getInfo()
+
+    def handle_message(self, message, bot_handler):
+        content = message['content']
+        topic = "test"
+        if 'subject' in message:
+            topic = message['subject']
+
+        response = game.beat(content, topic, self.merels_db)
```

Comment:
It seems like this logic will make all private conversations share the same game. You can make this work for private conversations by extracting the private conversation key from the `message` object.